finalPrep/modZeroHero.js

```diff
@@ -44,9 +54,21 @@
 //Test Cases
 var announcement = 'Never fear, the Courageous Curly Bracket is here!';
 var excuse = 'I think I forgot to lock up my 1992 Toyota Coralla. Be right back.';
-// assessSituation(99, announcement, excuse) > Should print - 'I think I forgot to lock up my 1992 Toyota Coralla. Be right back.'
-//assessSituation(21, announcement, excuse) > should print - 'Never fear, the Courageous Curly Bracket is here!'
-//assessSituation(3, announcement, excuse) > should print - "Meh. Hard pass."
+
+function assessSituation(dangerLevel,saveTheDay,badExcuse){
+  if (dangerLevel < 10){
+    console.log("Meh, Hard pass.");
+  }
+  else if(dangerLevel > 50){
+    console.log(badExcuse);
+  }else{
```

Comment:
Very nice job putting the most complex logic in your `else` block- that saves you quite a bit of typing, and is more readable.